surveys/apps.py

```diff
@@ -0,0 +1,54 @@
+from collections import OrderedDict
+
+from django.apps import AppConfig
+
+
+class SurveyConfig(AppConfig):
+    name = 'surveys'
+
+    # Complete dictionary of customisable data per field type.
+    # Has to be Ordered so we can guarantee consistent behaviour on things making
+    # use of it (dropdown fields etc).
+    field_type_data = OrderedDict((
+        ('free_text', {
+            'display_text': 'Character field (free text)',
+            'form_field_factory': None,
+            'serializer_field_factory': None,
+        }),
+        ('number', {
+            'display_text': 'Number',
+            'form_field_factory': None,
+            'serializer_field_factory': None,
+        }),
+        ('percentage', {
```

Comment:
These are drawn from 'the spec' for one of the projects we're using this in.  It'll be validated as a percentage (is the plan) but the slider is largely a UI decision.  So if anything the slider part is the suspect bit :P